server/mlabns/util/constants.py

```diff
@@ -40,8 +40,8 @@
 
 # URL to the privacy doc. All requests to http://mlab-ns.appspot.com/privacy
 # will be redirected to this URL.
-PRIVACY_DOC_URL = 'https://docs.google.com/a/google.com/document/d/1yQp7CcZngY6AfndoxvIbz8MzxcO7MpjZaj_VGFWe6Mo/pub'
+PRIVACY_DOC_URL = 'https://www.measurementlab.net/privacy/'
```

Comment:
i mentioned this to @collina a bit ago, but the privacy doc linked here was actually a different doc and focused on the mlab-ns privacy policy. I think we should review it and potentially add a section to the m-lab privacy page. so we can do this, but then we should make sure to reconcile the two other documents.

Reply:
The MLab-NS privacy doc should link to the MLab privacy doc, but they are not the same doc. Please keep this link the same.

Reply:
I will restore the original links for now.